integrationtests.py

```diff
@@ -806,6 +807,20 @@ def testHitsViaMpConcurrent(self):
                 self.assertEqual(stats.numCacheMisses(), 2)
                 self.assertEqual(stats.numCacheEntries(), 2)
 
+    def testOutput(self):
+        with cd(os.path.join(ASSETS_DIR, "parallel")), tempfile.TemporaryDirectory() as tempDir:
+            sources = [f for f in glob.glob("*.cpp")]
```

Comment:
This line is the same as

    sources = list(glob.glob("*.cpp"))

...but for me, `glob.glob` already returns a list, so maybe you can just do

    sources = glob.glob("*.cpp")